<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>0</y>
      <width>838</width>
      <height>522</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QGroupBox" name="SimulatorView">
       <property name="title">
        <string>CES</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QWidget" name="InfoDisplayGroup" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="BatteryBox">
               <item>
                <widget class="QLabel" name="ElectrodeLabel">
                 <property name="text">
                  <string>Electrodes: OFF</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QWidget" name="BatteryInfo" native="true">
                 <layout class="QHBoxLayout" name="_2">
                  <item>
                   <widget class="QLabel" name="BatteryLabel">
                    <property name="maximumSize">
                     <size>
                      <width>180</width>
                      <height>64</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Battery Label</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QProgressBar" name="BatteryBar">
                    <property name="minimumSize">
                     <size>
                      <width>180</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>180</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="value">
                     <number>100</number>
                    </property>
                    <property name="format">
                     <string>Battery %p%</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="PowerButton">
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>⏻</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QWidget" name="widget_6" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLCDNumber" name="TimerDisplay">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>48</height>
                   </size>
                  </property>
                  <property name="digitCount">
                   <number>8</number>
                  </property>
                  <property name="value" stdset="0">
                   <double>0.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="SettingsDisplayGroup">
               <property name="title">
                <string>Settings Display</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="FrequencyLabel">
                  <property name="text">
                   <string>Frequency Label</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="WaveformLabel">
                  <property name="text">
                   <string>WaveForm Label</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="AmpSelectorGroup" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QProgressBar" name="AmpProgressBar">
               <property name="maximum">
                <number>500</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="format">
                <string>%v amps</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="AmpDownButton">
               <property name="text">
                <string>Down-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="AmpUpButton">
               <property name="text">
                <string>Up+</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="SettingsBar">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <widget class="QGroupBox" name="TreatmentGroupBox">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>140</height>
               </size>
              </property>
              <property name="title">
               <string>Treatment</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="leftMargin">
                <number>8</number>
               </property>
               <property name="topMargin">
                <number>8</number>
               </property>
               <property name="rightMargin">
                <number>8</number>
               </property>
               <property name="bottomMargin">
                <number>8</number>
               </property>
               <item>
                <widget class="QPushButton" name="StartButton">
                 <property name="text">
                  <string>Start/Stop Treatment</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QPushButton" name="RecordButton">
                   <property name="text">
                    <string>Record</string>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="RecordIcon">
                   <property name="text">
                    <string>⬤</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QComboBox" name="RecordHistoryComboBox">
                 <property name="currentText">
                  <string notr="true">Default Treatment</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>Default Treatment</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="CycleGroup">
              <property name="title">
               <string>Cycles</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QRadioButton" name="Min20RadioButton">
                 <property name="text">
                  <string>20 min</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="Min40RadioButton">
                 <property name="text">
                  <string>40 min</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="Min60RadioButton">
                 <property name="text">
                  <string>60 min</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="FrequencyGroup">
              <property name="title">
               <string>Frequency </string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QRadioButton" name="Hz05RadioButton">
                 <property name="text">
                  <string>0.5Hz</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="Hz77RadioButton">
                 <property name="text">
                  <string>77Hz</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="Hz100RadioButton">
                 <property name="text">
                  <string>100Hz</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="WaveFormGroup">
              <property name="title">
               <string>WaveForm</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QRadioButton" name="AlphaRadioButtton">
                 <property name="text">
                  <string>Alpha</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="BetaRadioButton">
                 <property name="text">
                  <string>Beta</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="GammaRadioButton">
                 <property name="text">
                  <string>Gamma</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="AdminView">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>240</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Admin</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QGroupBox" name="ElectrodeGroup">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>180</height>
           </size>
          </property>
          <property name="title">
           <string>Electrode Connections</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QCheckBox" name="LeftEarCheck">
             <property name="text">
              <string>Left Ear</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="RightEarCheck">
             <property name="text">
              <string>Right Ear</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="KillButton">
          <property name="text">
           <string>Kill Button</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ChargeUpButton">
          <property name="text">
           <string>Charge Battery</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DecreaseBatteryButton">
          <property name="text">
           <string>-1% Battery</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DecreaseBatteryButton_2">
          <property name="text">
           <string>-10% Battery</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DecreaseTimeButton">
          <property name="text">
           <string>-30 seconds</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>857</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>AlphaToggle(bool)</slot>
  <slot>RecordToggle(bool)</slot>
 </slots>
</ui>
